README.md

```diff
@@ -1,12 +1,8 @@
-# java-dropwizard-swagger-jdbc
+# Agile Team 2 - Backend
 
 Database
 ---
-1. Create an empty database
-1. Run the following command from the base of your cloned directory to create the required database structure:
-```
-mysql --host=<localhost> --user=<your_username> --password=<your_password> <your_database_name> < employeesdb.sql
-```
+1. 
```

Comment:
Empty bullet?

Reply:
Just want to leave these blank as they will get done as part of the README ticket. Need to get DB setup too :)

Reply:
Updated now :)